css/80_app.css

```diff
@@ -3157,11 +3161,24 @@ div.full-screen > button:focus {
     background: #e8ebff;
 }
 
-.layer-list li.best > div.best {
+.layer-list li > div.stamp {
     padding: 5px;
     flex: 0 0 auto;
 }
 
+.layer-list li > div.imagery-icon {
+    margin: 5px;
+    max-width: 20px;
```

Comment:
This might just happen to do the right thing for [the abnormally wide icon](https://www.mcohio.org/_assets_/images/logo.png) specified by [this source](https://github.com/osmlab/editor-layer-index/blob/e1ec9cbf0146c8ca06d3fa6665901638a52ab92c/sources/north-america/us/oh/Montgomery_OH_2019.geojson?short_path=0de5cbe#L11).

Reply:
I forgot to remove this css from a first implementation. Now ([see below](https://github.com/openstreetmap/iD/pull/8826/files/79a7e88537cb275187bd63b04e9336dea8b62d22#diff-3f86fb5cebd04342575e756d440d18a0b6cfb131a3c26d7897ff1793f8b6d30bR3216-R3221)) the non square icons are just shrunk down to fit, which would totally not work in this case. :see_no_evil: